### PR TITLE
Fix EC2/Lightsail SD changelog entries for 2.39.x

### PR DESCRIPTION
**Pull request by juliusv**
As called out by @candlerb in https://github.com/prometheus/prometheus/pull/11326

Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,8 @@
 * [FEATURE] **experimental** TSDB: Add support for ingesting out-of-order samples. This is configured via `out_of_order_time_window` field in the config file; check config file docs for more info. #11075
 * [ENHANCEMENT] API: `/-/healthy` and `/-/ready` API calls now also respond to a `HEAD` request on top of existing `GET` support. #11160
 * [ENHANCEMENT] PuppetDB SD: Add `__meta_puppetdb_query` label. #11238
-* [ENHANCEMENT] AWS Lightsail SD: Add `__meta_ec2_region` label. #11326
-* [ENHANCEMENT] AWS EC2 SD: Add `____meta_lightsail_region` label. #11326
+* [ENHANCEMENT] AWS EC2 SD SD: Add `__meta_ec2_region` label. #11326
```
Comment (LeviHarrison) [inline CHANGELOG.md:8]:
```suggestion
* [ENHANCEMENT] AWS EC2 SD: Add `__meta_ec2_region` label. #11326
```

Comment (juliusv) [reply CHANGELOG.md:8]: Hah I copied the SD from below, where it is now missing. Will fix both, oops :)

Comment (LeviHarrison) [reply CHANGELOG.md]: Oh I missed that yeah :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,8 +5,8 @@
 * [FEATURE] **experimental** TSDB: Add support for ingesting out-of-order samples. This is configured via `out_of_order_time_window` field in the config file; check config file docs for more info. #11075
 * [ENHANCEMENT] API: `/-/healthy` and `/-/ready` API calls now also respond to a `HEAD` request on top of existing `GET` support. #11160
 * [ENHANCEMENT] PuppetDB SD: Add `__meta_puppetdb_query` label. #11238
-* [ENHANCEMENT] AWS Lightsail SD: Add `__meta_ec2_region` label. #11326
-* [ENHANCEMENT] AWS EC2 SD: Add `____meta_lightsail_region` label. #11326
+* [ENHANCEMENT] AWS EC2 SD: Add `__meta_ec2_region` label. #11326
+* [ENHANCEMENT] AWS Lightsail SD: Add `____meta_lightsail_region` label. #11326
```
Comment (LeviHarrison) [inline CHANGELOG.md:9]: Wait are the 4 underscore's intentional?

Comment (juliusv) [reply CHANGELOG.md:9]: Oh, they aren't! Although they were already there before... will fix that too!

**Review by LeviHarrison — APPROVED** [review]
Great, thanks.

**Comment by codesome** [comment]
Thanks!

**Comment by codesome** [comment]
Updated the tagged pre-release as well

**Pull request merged**